lighthouse-core/report/html/renderer/util.js

```diff
@@ -465,6 +465,13 @@ class Util {
 
     return lines.filter(line => lineNumbersToKeep.has(line.lineNumber));
   }
+
+  /**
+   * @param {string} categoryId
+   */
+  static isPluginCategory(categoryId) {
```

Comment:
adding onto [previous discussion](https://github.com/GoogleChrome/lighthouse/pull/8307#discussion_r275633709)....  yeahhhhhh this is feelin bad.  :/

my first inclination: `LH.Result.Category` should have an optional `plugin` property that uses the full plugin id.

@brendankenny we'll need your call on this.

Reply:
I guess it depends on what we mean by a "plugin category", if we want to badge basically anything that's not under the lighthouse repo's control (basically what's in the default or full configs, I guess), or just plugin plugins.

> my first inclination: LH.Result.Category should have an optional plugin property that uses the full plugin id.

Got you covered :)

If we want it to be strictly things passed in with `--plugins` (or `plugins` in the config), then (as @patrickhulce was saying in https://github.com/GoogleChrome/lighthouse/pull/8307#discussion_r275854898), the `categoryId` from any Lighthouse plugin has the same name as the plugin itself, and plugin names [have to start with `lighthouse-plugin-`](https://github.com/GoogleChrome/lighthouse/blob/53cea6d12883799627cbdbfe7ad8b30485e8bc13/lighthouse-core/config/config.js#L173), so checking that will be sufficient.

Reply:
> and plugin names have to start with lighthouse-plugin-, so checking that will be sufficient.

done

Reply:
> and plugin names have to start with lighthouse-plugin-

we are so smrt